docs/PowerAuth-SDK-for-Android.md

```diff
@@ -714,6 +882,107 @@ powerAuthSDK.fetchEncryptionKey(context, authentication, index, new IFetchEncryp
 })
 ```
 
+## Recovery Codes
+
+The recovery codes allows your users to recovery their activation in case that mobile device is lost or stolen. Before you start, please read [Activation Recovery](https://github.com/wultra/powerauth-crypto/blob/develop/docs/Activation-Recovery.md) document, available in our [powerauth-crypto](https://github.com/wultra/powerauth-crypto) repository.
```

Comment:
`The recovery codes allows your users to recovery their activation` -> `The recovery codes allow your users to recover their activation`